src/containers/collaborations/reducers/renameCollaborationDatabaseReducer.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * @flow
+ */
+
+import { Map } from 'immutable';
+import { CollaborationsApiActions } from 'lattice-sagas';
+import { RequestStates } from 'redux-reqseq';
+import type { SequenceAction } from 'redux-reqseq';
+
+import {
+  COLLABORATIONS,
+  ERROR,
+  REQUEST_STATE,
+  TITLE
+} from '../../../core/redux/constants';
+
+const { RENAME_COLLABORATION_DATABASE, renameCollaborationDatabase } = CollaborationsApiActions;
+
+export default function renameCollaborationDatabaseReducer(state :Map, action :SequenceAction) {
+  return renameCollaborationDatabase.reducer(state, action, {
+    REQUEST: () => state
+      .setIn([RENAME_COLLABORATION_DATABASE, REQUEST_STATE], RequestStates.PENDING)
+      .setIn([RENAME_COLLABORATION_DATABASE, action.id], action),
+    SUCCESS: () => {
+      const storedSeqAction = state.getIn([RENAME_COLLABORATION_DATABASE, action.id]);
+      if (storedSeqAction) {
+        const { collaborationId, name } = storedSeqAction.value;
+        return state
+          .setIn([COLLABORATIONS, collaborationId, TITLE], name)
```

Comment:
Renaming the database is independent of the title of the collaboration. You need to update the value in `DATABASE_DETAILS`.